### PR TITLE
update deep el vre assumption switch to match current data, and also …

### PR DESCRIPTION
**Pull request by cchrisgong**
…print a more helpful error message

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [X ] Bug fix 

## Checklist:

- [ X] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [ X] I performed a self-review of my own code
- [ X] I explained my changes within the PR, particularly in hard-to-understand areas
- [ X] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [ X] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [ X] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [ X] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [ X] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

### CODE REVIEW

**Comment by cchrisgong** [comment]
tagging @PhilippVerpoort in case deep el stuff

**Comment by cchrisgong** [comment]
@robertpietzcker @fschreyer do you know why under deep El switch cm_VRE_supply_assumptions = 1, incolearn is not modified, but under =2 and =3 they are? I thought from reading main.gms, that the floor costs of spv are supposed to be changed under all switch values. Thanks!

File: core/datainput.gms [hunk 1]
```diff
@@ -138,37 +138,37 @@ $include "./core/input/generisdata_trade.prn"
...
+    fm_dataglob("inco0","storspv") = 6470;
   );
 
-  if (fm_dataglob("incolearn","storspv") ne 5710,
-    abort "fm_dataglob('incolearn','storspv') is to be modified, but changed externally";
+  if (fm_dataglob("incolearn","storspv") ne 5440,
+    abort "input data in core/input/generisdata_tech.prn has been externally changed, so fm_dataglob('incolearn','storspv') specified here no longer matches the data there; code here needs to be updated to the new input data";
   else
-    fm_dataglob("incolearn","storspv") = 4240;
+    fm_dataglob("incolearn","storspv") = 4040;
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline core/datainput.gms:155]: The whole point of these checks is to ensure that ad-hoc modifications done via some so black Excel magic by @FalkoUeckerdt are not applied to data they were not based on.
This change just takes new, updated data, and declares those changes valid.  Unless Falko green-lights them or provides updated modifications, this cannot be merged.

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — CHANGES_REQUESTED** [review]

**Comment by cchrisgong** [comment]
@LaviniaBaumstark I tried to update the deepEl scenario csv, but my budget runs keep on failing because the cost data (before deepEl modifications) are outdated, in current REMIND version they are lower than they used to be when these deep El switches were implemented..

As you see here, I was trying to submit a bug fix a month ago, but Michaja complained about the format of the change (my bug fix would have created new links between new cost data and some extrapolated cost data based on new cost data and the ratio of the cost decline caused by deep El switch on)

I would like to fix this bug before I upload the new DeepEl csv

@gunnar-pik

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
I was not "complaining about the format" of your change.  I was questioning its content.

> my bug fix would have created new links between new cost data and some extrapolated cost data based on new cost data and the ratio of the cost decline caused by deep El switch on

To my understanding, @FalkoUeckerdt's process back then [was more involved](https://mattermost.pik-potsdam.de/rd3/pl/cyb16pjze3f95br7ue93icrw5e) than saying "well, 84 % of `inco0` and 74 % of `incolearn`.
So either get the OK from him that these numbers do check out, get new numbers from him, or get his process/Excel sheet and calculate the numbers.

P.S. I very much doubt that Gunnar receives Github notifications.  So if you want him to look at this, better send him a message.

**Comment by FalkoUeckerdt** [comment]
I would keep the original deepEl parameters (we were sufficiently optimistic 2 years ago), even though our default assumptions got more optimistic. I would thus suggest to not implement the deepEl numbers in relation or dependent on other (default) parameters, but as absolute values.

**Comment by cchrisgong** [comment]
hi, thanks both, I'm just back from vacation - will work on this this week @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — APPROVED** [review]

**Pull request merged**